project/patches/fbthrift-2021-11-29.patch

```diff
@@ -1,24 +0,0 @@
-diff -ur a/thrift/lib/cpp2/async/HeaderServerChannel.h b/thrift/lib/cpp2/async/HeaderServerChannel.h
---- a/thrift/lib/cpp2/async/HeaderServerChannel.h	2021-11-29 14:14:12.000000000 +0800
-+++ b/thrift/lib/cpp2/async/HeaderServerChannel.h	2022-10-13 16:22:03.142090462 +0800
-@@ -108,7 +108,7 @@
-     }
- 
-     bool isOneway() const override {
--      return header_->getSequenceNumber() == ONEWAY_REQUEST_ID;
-+      return header_ && header_->getSequenceNumber() == ONEWAY_REQUEST_ID;
```

Comment:
this may have some issue before, need check whether the new version need this patch